website/project/model.py

```diff
@@ -140,17 +140,22 @@ def validate_comment_reports(value, *args, **kwargs):
 
 class Comment(GuidStoredObject):
 
+    redirect_mode = 'proxy'
```

Comment:
`redirect_mode` on `GuidStoredObjects` no longer does anything. This can be removed.